.github/workflows/linters.yaml

```
@@ -56,3 +56,11 @@ jobs:
 
       - name: Lint Python files
         run: pylint $(git ls-files '*.py') -E
```

```suggestion
        run: pylint $(git ls-files '*.py')
```
here `-E` was throwing only errors and we want some warning too, to have better code quality.
And with removing `-E` I'll have to do otther changes, II'll update the pr soon

---

seems like whole lot of lints changes require

---

I tried to fix but difficult to fix all at once so till keeping this